ember-app/app/components/hb-column-crumb.js

```diff
@@ -6,10 +6,10 @@ var HbColumnCrumbComponent = Ember.Component.extend({
   classNameBindings: ["stateClass", "isSelected:active:inactive", "indexClass"],
   isSelected: function(){
     if(this.get("issue.data.state") === "closed"){
-      return this.get("visibleColumns.lastObject.data.name") === this.get("column.data.name");
+      return this.get("visibleColumns.lastObject.data.index") === this.get("column.data.index");
     }
-    return this.get("issue.data.current_state.name") === this.get("column.data.name");
-  }.property("issue.data.current_state.name", "column.data.name", "issue.data.state"),
+    return this.get("issue.columnIndex") === this.get("column.data.index");
+  }.property("issue.columnIndex", "column.data.index"),
```

Comment:
But I can see where you are going with that, I am going to reimplement using real array index.